client/src/actions/index.js

```diff
@@ -0,0 +1 @@
+import { TYPE1 } from './index';
```

Comment:
Shouldn't this import from a different file? from types.js?

Reply:
ohh yes right.... my bad... ill change it